tests/Tests/500__jupyterhub/test-jupyterlab-git.robot

```diff
@@ -0,0 +1,156 @@
+*** Settings ***
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library          OpenShiftCLI
+Library          DebugLibrary
+Suite Setup      Server Setup
+Suite Teardown   End Web Test
+
+
+*** Variables ***
+${REPO_URL} =    ****
+${DIR_NAME} =  Python
+${FILE_PATH} =   Python/file.ipynb
+${EMAIL_ID} =  user@gmail.com
+${NAME} =  user
+${COMMIT_MSG} =  commit msg2
+
+
+*** Test Cases ***
+# Have a remote repository configured
+# File -> save all changes
+# Click git -> simple staging
+# Click git -> push to remote
+
+Verify Pushing Project Changes Remote Repository
+    [Tags]  ODS-326
+    ...     Tier1
+    Push Some Changes to Repo    ${GITHUB_USER.USERNAME}  ${GITHUB_USER.TOKEN}   ${FILE_PATH}    ${REPO_URL}    ${COMMIT_MSG}
+
+Verify updating your project with changes from a git repository
+  [Tags]    ODS-324
+  ...       Tier1
+  Clone Git Repository And Open    ${REPO_URL}    ${FILE_PATH}
+  Sleep    1s
+  Open With JupyterLab Menu    File    New    Notebook
+  Sleep    2s
+  Maybe Close Popup
+  Close Other JupyterLab Tabs
+  Maybe Close Popup
+  Sleep    1
+  ${ouput1}=  Run Cell And Get Output    !git log --name-status HEAD^..HEAD
+  Log To Console    output 1.1 ${ouput1}
+  ${ouput1}=  Run Cell And Get Output    !git log --name-status HEAD^..HEAD
+  Log To Console    output 1.2 ${ouput1}
+  Log To Console    ---------
+
+  Add and Run JupyterLab Code Cell in Active Notebook  ! mkdir ../folder/
+  Log To Console    After !mkdir ../folder/
+  Add and Run JupyterLab Code Cell in Active Notebook  ! ls
+  Log To Console    After !ls
+  Sleep    5s
+  Open With JupyterLab Menu  File  Open from Path…
+  Input Text  xpath=//input[@placeholder="/path/relative/to/jlab/root"]  folder
+  Click Element  xpath://div[.="Open"]
+  Sleep    5s
+  #now do here some changes
```

Comment:
could you remove these comments (also line 60)? ideally, by reading the test case should be enought to understand what it is doing

Reply:
I have added comments for my understanding , I will remove them later